From: Adam Jackson <ajax@redhat.com>
Date: Sun, 28 Oct 2007 09:37:52 +0100
Subject: [PATCH] Don't backfill windows with bg=none

Disable backfilling of windows created with bg=none, which otherwise
would force a framebuffer readback.

Note that this patch can only be dropped if you verify that 

   https://bugzilla.redhat.com/show_bug.cgi?id=285991

is fixed (which it is in upstream git as of 

f98dfec79dadb70fa7bba84e7335f92b3a73dc02

---
--- a/composite/compalloc.c.spiffiffity	2006-03-13 16:59:55.000000000 -0500
+++ b/composite/compalloc.c	2006-04-12 16:37:50.000000000 -0400
@@ -478,6 +478,7 @@
      * Copy bits from the parent into the new pixmap so that it will
      * have "reasonable" contents in case for background None areas.
      */
+#if 0
     if (pGC)
     {
 	XID val = IncludeInferiors;
@@ -492,6 +493,7 @@
 			       w, h, 0, 0);
 	FreeScratchGC (pGC);
     }
+#endif
     return pPixmap;
 }
 
